binscripts/rvm-installer

```diff
@@ -219,7 +218,7 @@ __rvm_curl()
 )
 
 rvm_error()  { printf "ERROR: %b\n" "$*"; }
-__rvm_which(){   which "$@" || return $?; true; }
+__rvm_which(){ command -v "$@" 2>/dev/null; }
```

Comment:
please change to:
```bash
__rvm_which(){ builtin command -v "$@" 2>/dev/null || return $?; true; }
```
this will prevent tinkering with the command, and will fix problems with shell flags enforcing script fail on anyt command fail

also please note that `__rvm_which` is also used as the `which` to find files on `PATH` not functions, for example:
```bash
scripts/functions/osx-ssl-certs:95:  ssl_binary="${1:-$(__rvm_which openssl)}"
scripts/functions/env:140:  while local_rvm_path="$( __rvm_which rvm 2>/dev/null )"
```
this functionality has to be preserved, unfortunately neither `command -v` or `type -a` show you the path to a program if it's hidden with a function / alias

any ideas?